tagstudio/src/core/media_types.py

```diff
@@ -392,6 +392,11 @@ class MediaCategories:
         extensions=_IMAGE_VECTOR_SET,
         is_iana=False,
     )
+    IMAGE_RASTER_TYPES: MediaCategory = MediaCategory(
+        media_type=MediaType.IMAGE,
+        extensions=_IMAGE_SET,
+        is_iana=False,
+    )
     IMAGE_TYPES: MediaCategory = MediaCategory(
         media_type=MediaType.IMAGE,
         extensions=_IMAGE_SET | _IMAGE_RAW_SET | _IMAGE_VECTOR_SET,
```

Comment:
The `IMAGE_TYPES` media category should now include the new _IMAGE_RASTER_SET
```suggestion
        extensions=_IMAGE_SET | _IMAGE_RASTER_SET | _IMAGE_RAW_SET | _IMAGE_VECTOR_SET,
```